kotlin-vercel-ncc/src/jsMain/kotlin/vercel/NccOptions.kt

```diff
@@ -0,0 +1,78 @@
+package vercel
+
+import kotlinx.js.JsPlainObject
+
+@JsPlainObject
+external interface NccOptions {
+    /**
+     * provide a custom cache path as `String` or disable caching by setting to `false`
+     */
+    var cache: Any? /* string | boolean */
+
+    /**
+     * externals to leave as requires of the build
+     */
+    var externals: List<String>?
+
+    /**
+     * directory outside of which never to emit assets
+     */
+    var filterAssetBase: String?
+
+    /**
+     * default: false
+     */
+    var minify: Boolean?
+
+    /**
+     * default: false
+     */
+    var sourceMap: Boolean?
+
+    /**
+     * default: false
+     */
+    var assetBuilds: Boolean?
+
+    /**
+     * default: "../" treats sources as output-relative
+     * when outputting a sourcemap, automatically include
+     * source-map-support in the output file (increases output by 32kB).
+     */
+    var sourceMapBasePrefix: String?
+
+    /**
+     * default: true
+     */
+    var sourceMapRegister: Boolean?
+
+    /**
+     * default: "" does not generate a license file
+     */
+    var license: String?
+
+    /**
+     * default: false
+     */
+    var v8cache: Boolean?
+
+    /**
+     * default: false
+     */
+    var quiet: Boolean?
+
+    /**
+     * default: false
+     */
+    var debugLog: Boolean?
+
+    /**
+     * default: false
+     */
+    var transpileOnly: Boolean?
+
+    /**
+     * default: "es2015"
+     */
+    var target: String?
```

Comment:
Type - string union?

Reply:
With which values?
From the ncc code I only see it has to start with `es`.
The link it points to does not list targets.
I assume it is the ECMAScript versions, but from Wikipedia it seems every year a new one is released.

Reply:
For start `String` is fine